proposed/links.md

```diff
@@ -0,0 +1,141 @@
+- [RFC 5988](https://tools.ietf.org/html/rfc5988)
+- [Microformats Relations List](http://microformats.org/wiki/existing-rel-values#HTML5_link_type_extensions)
+- [IANA Link Relations Registry](http://www.iana.org/assignments/link-relations/link-relations.xhtml)
+
+
+## 1. Specification
+
+### 1.1 
+
+A Hypermedia Link consists of, at minimum:
+- A URI representing the target resource being referenced.
+- A relationship defining how the target resource relates to the source.
+
+Various other attributes of the Link may exist, depending on the format used. As additional attributes 
+are not well-standardized or universal, this specification does not seek to standardize them.
+
+
+## 2. Package
+
+The interfaces and classes described are provided as part of the
+[psr/link](https://packagist.org/packages/psr/link) package.
+
+## 3. Interfaces
+
+### 3.1 `Psr\Http\Link\LinkInterface`
+
+```php
+<?php
+namespace Psr\Http\Link;
+
+/**
+ * 
+ */
+interface LinkInterface
+{
+    /**
+     * Returns the target of the link.
+     *
+     * The target must be a URI or a Relative URI reference.
+     *
+     * @return string
+     */
+    public function getHref();
+
+    /**
+     * Returns the relationship type(s) of the link.
+     *
+     * This method returns 0 or more relationship types for a link, expressed
+     * as an array of strings.
+     *
+     * The returned values should be either a simple keyword or an absolute
+     * URI. In case a simple keyword is used, it should match one from the
+     * IANA registry at:
+     *
+     * http://www.iana.org/assignments/link-relations/link-relations.xhtml
+     *
+     * Optionally the microformats.org registry may be used, but this may not
+     * be valid in every context:
+     *
+     * http://microformats.org/wiki/existing-rel-values
+     *
+     * Private relationship types should always be an absolute URI.
+     *
+     * @return string[]
+     */
+    public function getRel();
+
+    /**
+     * Returns a list of attributes that describe the target URI.
+     *
+     * The list should be specified as a key-value list.
+     *
+     * There is no formal registry of the values that are allowed here, and
+     * validity of values is dependant on context.
+     *
+     * Common values are 'hreflang', 'title', and 'type'. Implementors
+     * embedding a serialized version of a link are responsible for only
+     * encoding the values they support.
+     *
+     * Any value that appears that is not valid in the context in which it is
+     * used should be ignored.
+     *
+     * Some attributes, (commonly hreflang) may appear more than once in their
+     * context. Attributes such as those may be specified as an array of
+     * strings.
+     *
+     * @return array
+     */
+    public function getAttributes();
+}
+```
+
+#### 3.2.1 `Psr\Http\Message\LinkableInterface`
+
+```php
+<?php
+namespace Psr\Http\Link;
+
+/**
+ * 
+ */
+interface LinkableInterface
```

Comment:
@quimcalpe Best place to discuss this is the mailing list: https://groups.google.com/forum/#!topic/php-fig/SFJK_Ynivu4